tests/lib.sh

```diff
@@ -26,8 +26,11 @@ blake2b_lock_hash () {
 }
 
 check_signature () {
-  r_val=$(head -c64 <<<"$2")
-  s_val=$(tail -c+65 <<<"$2" | head -c64)
+  sig=$(tr -d -c '[:xdigit:]' <<<"$3")
+  echo "$sig" >> sigval
```

Comment:
I think we may want to remove these temp files. It’s good for debugging but ends up littering your directory if you run the test suite outside of a nix builder.

Reply:
Whoops, yes.